apps/server/apis/grpcapi/internal.go

```diff
@@ -178,8 +159,21 @@ func (s *InternalServer) Register(request *protos.RegisterRequest, server protos
 	// to the ServiceName (so we can get all active tails for that service)
 	go s.sendActiveTails(server.Context(), ch, request)
 
-	// Send ephemeral schema inference pipeline for each announced audience
-	go s.sendInferSchemaPipelines(server.Context(), ch, request.SessionId)
+	///////////////////////////////////////////////////////////////////////////
+	//
+	// IMPORTANT: Schema inference pipeline is injected in THREE places:
+	//
+	// 	1. ✅In broadcast handler for SetPipelinesRequest (when pipelines are
+	//	   attached/detached for live clients)
+	//
+	// 	2. ✅In internal.GetSetPipelinesCommand() which is called when a client
+	//	   registers for the first time
+	//
+	//  3. In internal.NewAudience() which is called by SDKs when they announce a
```

Comment:
Update this - it occurs in a lot of different places - UpdatePipeline, PausePipeline, ResumePipeline, etc.